src/main/api/2/studio-api-2.yaml

```diff
@@ -3488,6 +3488,52 @@ paths:
         '403':
           $ref: '#/components/responses/Forbidden'
 
+  /configuration/content-type/preview_image:
+    get:
+      tags:
+        - configuration
+      summary: Get preview image of a given content-type
+      operationId: getContentTypePreviewImage
```

Comment:
Add `description` and indicate permissions.

Something like:
`Required permission "read_configuration"`

Reply:
Updated. Thank you!